lib/helpers.js

```diff
@@ -0,0 +1,36 @@
+const assert = (condition, msg = 'assertion failed') => {
```

Comment:
A lot of the diff in this PR is from moving files around to have better names. Many didn't change much or at all, like this file.